modules/ClinikalAPI/src/ClinikalAPI/Controller/PdfBaseController.php

```diff
@@ -118,4 +156,61 @@ public function createBase64Pdf($fileName,$bodyPath,$headerPath, $footerPath,$he
 
         return $pdfEncoded;
     }
+
+    public function getTitleOfOptionFromListTable($list,$option){
+        $listsTable= $this->container->get(ListsTable::class);
+        $title = $listsTable->getSpecificTitle($list,$option);
+        return $title;
+    }
+
+    public function getReasonCodesTitles($list,$codes){
+        $listsTable= $this->container->get(ListsTable::class);
+        $titles = $listsTable->getTitles($list,$codes);
+        return $titles;
+    }
+    public function getReasonCodes($encounter_id){
+        $reasonCode= $this->container->get(EncounterReasonCodeMapTable::class);
+        $reasonCodes = $reasonCode->fatchAllByEID($encounter_id,true);
+        return $reasonCodes;
+    }
+
+
+    public function getMedicalAdmissionQData($qid,$class){
```

Comment:
All the next functions that fetch data from form_medical_admission_questionnaire and form_diagnosis_and_recommendations_questionnaire need to move to emergency module

Reply:
resolved some function that concern to emergency where moved into emergency but I think you will continue using them also in other platforms since they are getPregnancyState,getFindings,getDiagnostics but time will tell.